### PR TITLE
`fk key from-gpg`: filter out keys already imported

### PR DESCRIPTION
**Pull request by paulfurley**

### CODE REVIEW

**Comment by paulfurley** [comment]
Rebasing

File: fluidkeys/main.go [hunk 1]
```diff
@@ -105,23 +106,39 @@ func keySubcommand(args docopt.Opts) exitCode {
...
-	fmt.Printf(formatListedKeysForImportingFromGpg(secretKeys))
-	keyToImport := promptForKeyToImportFromGpg(secretKeys)
 
-	fluidkeysDirectory, err := getFluidkeysDirectory()
+	var availableKeys []gpgwrapper.SecretKeyListing
+
+	allGpgKeys, err := gpg.ListSecretKeys()
 	if err != nil {
-		fmt.Printf("Failed to get fluidkeys directory")
+		fmt.Errorf("Error getting secret keys from GPG: %v", err)
```
Comment (idrysdale) [inline fluidkeys/main.go]: Why's this `Errorf`, and above `Printf`?

**Comment by idrysdale** [comment]
Cool. That looks good to me. We're still switching between 'importing', 'linking' and 'managing' keys in Fluidkeys but I think that's something for discussion for outside of this pull request!

**Comment by paulfurley** [comment]
Agreed, let's follow up with a PR that unifies the language.

File: fluidkeys/main.go (hunk 1 above)
Comment (paulfurley) [reply fluidkeys/main.go]: because i'm a pillock

**Comment by paulfurley** [comment]
I'm rebasing this now

**Pull request merged**